integration

```diff
@@ -600,6 +600,7 @@
   "markvader/sonic",
   "marotoweb/home-assistant-vacuum-viomise",
   "marq24/ha-senec-v3",
+  "marq24/ha-tibber-pulse-local",
```

Comment:
Your value for the `country` key in `hacs.json` is not valid in the current release.

<https://hacs.xyz/docs/publish/start/#hacsjson>
> Two character country code in ISO 3166-1 alpha-2 format.

Reply:
resolved in master

Reply:
You need to make a new release.

Reply:
2023.8.9 released - just for HACS only...